V2/tests/api/azure/project_azure_endpoints_test.py

```diff
@@ -0,0 +1,254 @@
+
+
+def test__delete_projects_azure_endpoint__returns_a_status_code_404__when_no_found_a_project_to_delete(
+):
+    req = func.HttpRequest(
+        method="DELETE",
+        body=None,
+        url=PROJECT_URL,
+        route_params={"id": Faker().pyint()},
+    )
+
+    response = azure_projects._delete_project.delete_project(req)
+
+    assert response.status_code == HTTPStatus.NOT_FOUND
+    assert response.get_body() == b"Not found"
+
+
+def test__update_project_azure_endpoint__returns_an_project__when_found_a_project_to_update(
+    insert_project
+):
+    inserted_project = insert_project().__dict__
+
+    project_body = {"description": Faker().sentence()}
+    req = func.HttpRequest(
+        method='PUT',
+        body=json.dumps(project_body).encode("utf-8"),
+        url=PROJECT_URL,
+        route_params={"id": inserted_project["id"]},
+    )
+
+    response = azure_projects._update_project.update_project(req)
+    activitiy_json_data = response.get_body().decode("utf-8")
+    inserted_project.update(project_body)
+
+    assert response.status_code == HTTPStatus.OK
+    assert activitiy_json_data == json.dumps(inserted_project)
+
+
+def test__update_projects_azure_endpoint__returns_a_status_code_404__when_no_found_a_project_to_update(
+    project_factory
+):
+    project_body = project_factory().__dict__
+
+    req = func.HttpRequest(
+        method="PUT",
+        body=json.dumps(project_body).encode("utf-8"),
+        url=PROJECT_URL,
+        route_params={"id": Faker().pyint()},
+    )
+
+    response = azure_projects._update_project.update_project(req)
+
+    assert response.status_code == HTTPStatus.NOT_FOUND
+    assert response.get_body() == b"Not found"
+
+
+def test__update_projects_azure_endpoint__returns_a_status_code_400__when_recive_an_incorrect_body(
+):
+    project_body = Faker().pydict(5, True, str)
+    req = func.HttpRequest(
+        method="PUT",
+        body=json.dumps(project_body).encode("utf-8"),
+        url=PROJECT_URL,
+        route_params={"id": Faker().pyint()},
+    )
+
+    response = azure_projects._update_project.update_project(req)
+
+    assert response.status_code == HTTPStatus.BAD_REQUEST
+    assert response.get_body() == b"Incorrect body"
+
+
+def test__update_projects_azure_endpoint__returns_a_status_code_400__when_project_recive_invalid_id(
+):
+    req = func.HttpRequest(
+        method="PUT",
+        body=None,
+        url=PROJECT_URL,
+        route_params={"id": "invalid id"},
+    )
+
+    response = azure_projects._update_project.update_project(req)
+
+    assert response.status_code == HTTPStatus.BAD_REQUEST
+    assert response.get_body() == b"Invalid Format ID"
+
+
+def test__project_azure_endpoint__creates_an_project__when_project_has_all_attributes(
+    test_db, project_factory, insert_customer, customer_factory
+):
+    insert_customer(customer_factory(), test_db)
+    project_body = project_factory().__dict__
+    body = json.dumps(project_body).encode("utf-8")
```

Comment:
The same format of the previous tests should be followed, where this line is made within the request. Also do that in the other cases